app/src/views/features/rules/RuleEditor/components/Header/ActionButtons/Status/index.js

```diff
@@ -38,6 +40,20 @@ const Status = ({ isDisabled = false, location, isRuleEditorModal }) => {
       status: newValue,
     };
 
+    const isCreateMode = location.pathname.indexOf("create") !== -1;
+
+    if (ruleData.isSample && !isCreateMode) {
```

Comment:
Add optional chaining. Also please verify for other instance wherever needed.

Reply:
Not needed in this case